web-client/src/app/components/pay-amount-confirm/pay-amount-confirm.component.html

```diff
@@ -1 +1,36 @@
-<p>pay-amount-confirm works!</p>
+<ion-grid>
+  <ion-row>
+    <ion-col>
+      <app-pay-from-to
+        [name]="name"
+        [balance]="balance"
+        [receiverAddress]="receiverAddress"
+      ></app-pay-from-to>
+    </ion-col>
+  </ion-row>
+
+  <ion-row>
+    <ion-col>
+      <!--
+        NOTE(Pi): <app-pay-from-to> is inside an <ion-card>,
+        which gives it some extra margin compared to <app-pay-amount-form>,
+        which isn't inside a margined contained.
```

Comment:
Typo here: "Margined container"

Reply:
Oh, this was actually intentional. The phrasing's a bit unusual, but it's apparently correct, and I can't really think of a better phrasing that isn't more cumbersome.